packages/js/product-editor/src/components/variations-table/downloads-menu-item/downloads-menu-item.tsx

```diff
@@ -139,7 +139,10 @@ export function DownloadsMenuItem( {
 
 	return (
 		<Dropdown
-			position="middle right"
+			// @ts-expect-error missing prop in types.
+			popoverProps={ {
+				placement: 'right-start',
```

Comment:
I think `position="middle right"` is equivalent to `placement: 'right'`?

Reply:
Yeah, although I updated it to use `right-start` instead, it seemed better:
`right`:
<img width="497" alt="Screenshot 2023-12-12 at 3 31 58 PM" src="https://github.com/woocommerce/woocommerce/assets/2240960/dc45cdba-6e85-4a70-9d2b-7946527657d1">


`right-start`:
<img width="501" alt="Screenshot 2023-12-12 at 3 32 08 PM" src="https://github.com/woocommerce/woocommerce/assets/2240960/55fff77f-966c-4168-9a17-ea68eab45873">